code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -115,7 +138,23 @@
 /obj/item/clothing/gloves/rig/ce
 	name = "insulated gloves"
 	siemens_coefficient = 0
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+	armor = list(melee = 50, bullet = 40, laser = 40,energy = 25, bomb = 60, bio = 100, rad = 100)
```

Comment:
All armor statistics like max_heat_protection_temperature and armor values are inherited from the weapon/rig item. They have no effect when defined in a rig piece except for siemens_coefficient. See line 141 in rig.dm.